compiler/q-implant/src/QImplant.cpp

```diff
@@ -251,6 +271,50 @@ void QImplant::write(loco::Graph *g)
     }
   }
 
+  std::set<luci::CircleNode *> visit;
+  std::deque<luci::CircleNode *> que;
```

Comment:
please rename `visit`, `que` to some meaningful word.

Reply:
As suggestion about using `loco::postorder_traversal` by @jinevening,
replaced it to `loco::postorder_traversal`